.github/CODEOWNERS

```diff
@@ -2,12 +2,12 @@
 * @awslabs/aws-sdk-js-team
 
 # These libraries are only intended for use with the SSDK.
-smithy-typescript-ssdk-libs/* @adamthom-amzn @gosar @JordonPhillips
+smithy-typescript-ssdk-libs/* @adamthom-amzn @gosar @JordonPhillips @awslabs/smithy
```

Comment:
Can we remove the individuals while making this change? @adamthom-amzn @gosar and @JordonPhillips are all part of the Smithy team.